src/construction/old_construction.py

```diff
@@ -0,0 +1,282 @@
+"""DEPRECATED MODULE DO NOT USE."""
```

Comment:
Do you think we should keep this script in the code? Do you think we might use it later?